### PR TITLE
feat: generate SQL models diagram with petisco-dev

### PR DESCRIPTION
**Pull request by fgsalomon**
Copied from https://github.com/Dicklesworthstone/sqlalchemy_data_model_visualizer

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -62,5 +62,6 @@
         "elastic-apm": ["elastic-apm<7.0.0,>=6.15.1"],
         "fastapi": ["fastapi<1.0.0,>=0.100"],
         "rich": ["rich"],
+        "dev": ["graphviz"],
```
Comment (acostapazo) [inline setup.py:65]: I've modified this to use `dev`. This generic name could be used for different requirements as `graphviz` in this nice cli feature.

File: petisco/cli/petisco_dev.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
+from __future__ import annotations
```
Comment (acostapazo) [inline petisco/cli/petisco_dev.py:1]: Added to use `list`instead `typing.List` for older Python versions.

**Comment by acostapazo** [comment]
Awesome tool! Congrats!

I've tried with one of our services installing locally with:

```
pip install /m//petisco/source/code/path
```

Then I test this warning print with

```
(venv) ➜  my-app git:(main) ✗ petisco-dev --sql-models-diagram                                                                    

⚠️ If you want to generate a SVG diagram of Petisco SQL models you need to install graphviz. Try to install graphviz or just install petisco[graphviz]
```

So, I installed `dev` extra with:

```
(venv) ➜  my-app git:(main) ✗ pip install "petisco[dev]" 
``` 

Then I tried again a got the following error



```
(venv) ➜  my-app git:(main) ✗ petisco-dev --sql-models-diagram                                                                    
.
.
.
graphviz.backend.execute.ExecutableNotFound: failed to execute PosixPath('dot'), make sure the Graphviz executables are on your systems' PATH

```

**Comment by fgsalomon** [comment]
> Awesome tool! Congrats!
> 
> I've tried with one of our services installing locally with:
> 
> ```
> pip install /m//petisco/source/code/path
> ```
> 
> Then I test this warning print with
> 
> ```
> (venv) ➜  my-app git:(main) ✗ petisco-dev --sql-models-diagram                                                                    
> 
> ⚠️ If you want to generate a SVG diagram of Petisco SQL models you need to install graphviz. Try to install graphviz or just install petisco[graphviz]
> ```
> 
> So, I installed `dev` extra with:
> 
> ```
> (venv) ➜  my-app git:(main) ✗ pip install "petisco[dev]" 
> ```
> 
> Then I tried again a got the following error
> 
> ```
> (venv) ➜  my-app git:(main) ✗ petisco-dev --sql-models-diagram                                                                    
> .
> .
> .
> graphviz.backend.execute.ExecutableNotFound: failed to execute PosixPath('dot'), make sure the Graphviz executables are on your systems' PATH
> ```

Yeah, I didn't know how to solve this. This feature need the Python library and a OS library. In Mac you can install with `brew install graphviz".

File: setup.py (hunk 1 above)
Comment (fgsalomon) [reply setup.py:65]: Yes, much better

**Comment by acostapazo** [comment]
Understood.

Proposed solution

```
(venv) ➜  new-onboarding git:(main) ✗ petisco-dev --sql-models-diagram 
failed to execute PosixPath('dot'), make sure the Graphviz executables are on your systems' PATH

⚠️ The Graphviz Python package has already been installed, but it requires an executable to be available.
🍎 If you're using macOS, please ensure that the Graphviz executable is installed. You can do this by running 'brew install graphviz' in your terminal.
```

**Pull request merged**